app/controllers/api/v1/appointments_controller.rb

```diff
@@ -1,4 +1,4 @@
-class Api::V1::AppointmentsController < Api::Current::AppointmentsController
+class Api::V1::AppointmentsController < Api::V2::AppointmentsController
```

Comment:
The generator can be updated to do this automatically in the future.
Created [a story](https://www.pivotaltracker.com/story/show/165438812) for this.